README.md

```diff
@@ -85,25 +101,21 @@ variables are set on your machine.
 For example, to run pubsub tests against go-ipfs and js-ipfs revision defined
 in `./scripts/custom-runtime.sh`, one can:
 
-```
-export IPFS_GO_EXEC=/tmp/go-ipfs/cmd/ipfs/ipfs
-export IPFS_JS_EXEC=/tmp/js-ipfs/packages/ipfs/src/cli.js
-export IPFS_JS_MODULE=/tmp/js-ipfs/packages/ipfs/dist/cjs/src/index.js
-export IPFS_JS_HTTP_MODULE=/tmp/js-ipfs/packages/ipfs-http-client/dist/cjs/src/index.js
-./scripts/custom-runtime.sh
-node bin/ipfs-interop.js -- -t node --grep "pubsub"
-```
+    export IPFS_GO_EXEC=/tmp/go-ipfs/cmd/ipfs/ipfs
+    export IPFS_JS_EXEC=/tmp/js-ipfs/packages/ipfs/src/cli.js
+    export IPFS_JS_MODULE=/tmp/js-ipfs/packages/ipfs/dist/cjs/src/index.js
+    export IPFS_JS_HTTP_MODULE=/tmp/js-ipfs/packages/ipfs-http-client/dist/cjs/src/index.js
```

Comment:
should it be `dist/cjs` with this PR or `dist/esm` ?

Reply:
It should really be `export IPFS_JS_MODULE=/tmp/js-ipfs/packages/ipfs/src/index.js` as the source is just in `src` again.

Reply:
Have updated it